lang-guide/chapters/filters/select-get.md

```diff
@@ -1,3 +1,33 @@
 # Understanding the difference between `get` and `select`
 
-TODO
+### Get
+
+Extract data using a cell path.
+
+This is equivalent to using the cell path access syntax: `$env.OS` is the same as `$env | get OS`.
+
+If multiple cell paths are given, this will produce a list of values.
+
+```nu
+'{"name":"Alice","job":"Engineer"}'
+| from json
+| get name
+| describe
+# => string
+```
+
+### Select
+
+Select only these columns or rows from the input. Opposite of `reject`.
+
+This differs from `get` in that, rather than accessing the given value in the data structure, it removes all non-selected values from the structure.
```

Comment:
This statement is a little confusing. I always thought of `get` and `select` like this.
- `get` gets the value out of the structure it is in
- `select` selects the value maintaining the structure it is in

Reply:
That makes sense.  I just grabbed the language from the implementation, `scope commands | where name == select | get extra_description.0`.  I didn't realize it was already documented in the Book, so I will close this PR.